lua/octo/window.lua

```diff
@@ -25,78 +24,84 @@ function M.create_centered_float(opts)
       max_line = math.max(#line, max_line)
     end
   end
-
   -- calculate window height/width
   local width, height
   if max_line > 0 then
     -- pre-defined content
-    width = math.min(vim_width * 0.9, max_line)
+    width = math.min(vim_width * 0.9, max_line + 4)
```

Comment:
Where do these "random" numbers from? It will be hard to come back to this code in 2 years and remember what they where for. This either results into the scenario that code remains untouched because developers are afraid to touch it (especially if there are no tests for it) or the developer just deletes it, eventually breaks something which goes undetected (no tests) or he spends many time in research what this was originally for.
Either try to express this number or the math expression as code or (way less preferable) as comment.